accelerometer/summariseEpoch.py

```diff
@@ -218,11 +219,14 @@ def check_daylight_savings_crossover(e, startTime, endTime, summary,
         for t in localTime._utc_transition_times:
             if t>startTime:
                 transition = t
+                transition = transition - pd.DateOffset(seconds = 1)
+                transition = utc_tz.localize(transition)
+                transition = localTime.normalize(transition)
+                transition = transition.replace(tzinfo = None)
```

Comment:
in the last line, the output is almost what we wanted, meaning the value of 1:59:59am is correct (1 second apart from 2am crossover), but we want to remove the timezone association, so that it can be used with comparison / calculation with e.index timestamps

calling "replace" with tzinfo=None would do exactly that.

the outcome of this line, for US/eastern:
1. spring crossover: 1:59:59am US/eastern -> 1:59:59am
1. fall crossover: 1:59:59am US/eastern -> 1:59:59am